### PR TITLE
TextBox: Add a Getting Started tutorial

### PR DESCRIPTION
**Pull request by vladaskorohodova**

### CODE REVIEW

File: api-reference/10 UI Components/dxFilterBuilder/1 Configuration/onEditorPrepared.md [hunk 1]
```diff
@@ -53,6 +53,6 @@ The editor's value.
 The editor's width.
```
Comment (albertov05) [inline api-reference/10 UI Components/dxFilterBuilder/1 Configuration/onEditorPrepared.md:54]: Can you update line 47 above?

Line 47:
`Gets and sets the delay between when a user stops typing the field's value and when it is applied.`

I'd like to make "it" more specific. Can we say the following?

`Gets and sets the delay between when a user stops typing the field's value and when the change is applied.`

File: api-reference/10 UI Components/dxFilterBuilder/1 Configuration/onEditorPreparing.md [hunk 2]
```diff
@@ -61,9 +61,9 @@ Gets and sets the delay between when a user stops typing the field value and whe
 The editor's width.
```
Comment (albertov05) [inline api-reference/10 UI Components/dxFilterBuilder/1 Configuration/onEditorPreparing.md:62]: Can you update line 55 above?

Line 55:
`Gets and sets the delay between when a user stops typing the field value and when it is applied.`

Can we make "it" more specific?

`Gets and sets the delay between when a user stops typing the field value and when the change is applied.`

File: concepts/05 UI Components/TextBox/00 Getting Started with TextBox/00 Getting Started with TextBox.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+#include tutorials-intro-installationnote
+
+TextBox is a component that allows users to enter and edit a single line of text.
+
+This tutorial shows how to add a TextBox to your application and configure its core features.
+
+<div class="simulator-desktop-container" data-view="/Content/Applications/21_2/GettingStartedWith/TextBox/index.html, /Content/Applications/21_2/GettingStartedWith/TextBox/index.js, /Content/Applications/21_2/GettingStartedWith/TextBox/index.css"></div>
+
+Each section in this tutorial covers a single configuration step. You can also find the full code in the following GitHub repository: <a href="https://github.com/DevExpress-Examples/getting-started-with-devextreme-textbox" target="_blank">getting-started-with-textbox</a>.
```
Comment (albertov05) [inline concepts/05 UI Components/TextBox/00 Getting Started with TextBox/00 Getting Started with TextBox.md]:
```suggestion
Each section in this tutorial describes a single configuration step. You can also find the full code in the following GitHub repository: <a href="https://github.com/DevExpress-Examples/getting-started-with-devextreme-textbox" target="_blank">getting-started-with-textbox</a>.
```

File: concepts/05 UI Components/TextArea/00 Overview.md [hunk 4]
```diff
@@ -228,6 +228,6 @@ If an end user should _not_ be able to edit the text in the TextArea, assign **t
 - [TextArea - Handle the Keyboard Events](/concepts/05%20UI%20Components/TextArea/13%20Handle%20the%20Keyboard%20Events.md '/Documentation/Guide/UI_Components/TextArea/Handle_the_Keyboard_Events/')
 - [TextArea - Limit the Text Length](/concepts/05%20UI%20Components/TextArea/15%20Limit%20the%20Text%20Length.md '/Documentation/Guide/UI_Components/TextArea/Limit_the_Text_Length/')
 - [TextArea API Reference](/api-reference/10%20UI%20Components/dxTextArea '/Documentation/ApiReference/UI_Components/dxTextArea/')
-- [TextBox](/concepts/05%20UI%20Components/TextBox/00%20Overview.md '/Documentation/Guide/UI_Components/TextBox/Overview/')
+- [TextBox](/Documentation/ApiReference/UI_Components/dxTextBox/)
```
Comment (albertov05) [inline concepts/05 UI Components/TextArea/00 Overview.md:232]: Can you update the following lines?

Line 1:
`The TextArea is a UI component that enables a user to enter and edit a multi-line text.`

Please change "edit a multi-line text" to "edit multi-line text".

Line 78:
`By default, the TextArea checks the entered text for spelling errors.`

Please rewrite without using "By default".

`To disable this feature, assign **false** to the **spellcheck** property.`

Can we say the following instead?

`To disable this feature, set the **spellcheck** property to **false**.`

File: concepts/05 UI Components/TextBox/00 Getting Started with TextBox/10 Configure the Input Mode.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
+Specify the TextBox [mode](/api-reference/10%20UI%20Components/dxTextBox/1%20Configuration/mode.md '/Documentation/ApiReference/UI_Components/dxTextBox/Configuration/#mode') property to allow users to type in a specific text type. Set this property to *"email"*, *"tel"* or *"url"* to affect the set of keyboard buttons that a mobile device shows when the UI component gets focused. When you set the **mode** property to *"search"*, the TextBox contains the **Clear** button, which empties its contents. When you set the **mode** property to *"password"*, the TextBox hides input characters behind asterisks.
```
Comment (albertov05) [inline concepts/05 UI Components/TextBox/00 Getting Started with TextBox/10 Configure the Input Mode.md]: `Set this property to *"email"*, *"tel"* or *"url"* to affect the set of keyboard buttons that a mobile device shows when the UI component gets focused.`

I'd like to make the word "affect" more specific. Can we say "specify"?
Also, please add a comma after `*"tel"*`.
Let's also change "gets focused" to "is focused".

`Set this property to *"email"*, *"tel"*, or *"url"* to specify the set of keyboard buttons that a mobile device shows when the UI component is focused.`

File: concepts/05 UI Components/TextBox/00 Getting Started with TextBox/15 Specify a Label or Placeholder.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
+Use the [placeholder](/Documentation/ApiReference/UI_Components/dxTextBox/Configuration/#placeholder) property to give users a hint about what they should type in the TextBox. You can also use the [label](/Documentation/ApiReference/UI_Components/dxTextBox/Configuration/#label) property for this purpose. If you specify the **label** property, set the [labelMode](/Documentation/ApiReference/UI_Components/dxTextBox/Configuration/#labelMode) property to one of the following values:
+
+- *"static"*    
+The component displays the label above the input field.
+
+- *"floating"*    
+The component uses label as a placeholder, but when the editor gets focus, the label moves to the position above the input field.
```
Comment (albertov05) [inline concepts/05 UI Components/TextBox/00 Getting Started with TextBox/15 Specify a Label or Placeholder.md]:
```suggestion
The component uses the label as a placeholder, but when the editor gets focus, the label moves to the position above the input field.
```

File: concepts/05 UI Components/TextBox/00 Getting Started with TextBox/30 Handle the Keyboard Events.md [hunk 7]
```diff
@@ -0,0 +1,89 @@
+The TextBox raises three keyboard events: [keyDown](/api-reference/10%20UI%20Components/dxTextEditor/4%20Events/keyDown.md '/Documentation/ApiReference/UI_Components/dxTextBox/Events/#keyDown'), [keyUp](/api-reference/10%20UI%20Components/dxTextEditor/4%20Events/keyUp.md '/Documentation/ApiReference/UI_Components/dxTextBox/Events/#keyUp') and [enterKey](/api-reference/10%20UI%20Components/dxTextEditor/4%20Events/enterKey.md '/Documentation/ApiReference/UI_Components/dxTextBox/Events/#enterKey'). Handle these events to access the original keyboard events.
```
Comment (albertov05) [inline concepts/05 UI Components/TextBox/00 Getting Started with TextBox/30 Handle the Keyboard Events.md]:
```suggestion
The TextBox raises three keyboard events: [keyDown](/api-reference/10%20UI%20Components/dxTextEditor/4%20Events/keyDown.md '/Documentation/ApiReference/UI_Components/dxTextBox/Events/#keyDown'), [keyUp](/api-reference/10%20UI%20Components/dxTextEditor/4%20Events/keyUp.md '/Documentation/ApiReference/UI_Components/dxTextBox/Events/#keyUp'), and [enterKey](/api-reference/10%20UI%20Components/dxTextEditor/4%20Events/enterKey.md '/Documentation/ApiReference/UI_Components/dxTextBox/Events/#enterKey'). Handle these events to access the original keyboard events.
```

File: concepts/05 UI Components/TextBox/00 Getting Started with TextBox/30 Handle the Keyboard Events.md [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+
+In this tutorial, users can see the TextBox [value](/Documentation/ApiReference/UI_Components/dxTextBox/Configuration/#value) in the browser console after they hit the Enter key.
```
Comment (albertov05) [inline concepts/05 UI Components/TextBox/00 Getting Started with TextBox/30 Handle the Keyboard Events.md]:
```suggestion
In this tutorial, users can see the TextBox [value](/Documentation/ApiReference/UI_Components/dxTextBox/Configuration/#value) in the browser console after they press the Enter key.
```

File: api-reference/10 UI Components/dxFilterBuilder/1 Configuration/onEditorPrepared.md (hunk 1 above)
Comment (vladaskorohodova) [reply api-reference/10 UI Components/dxFilterBuilder/1 Configuration/onEditorPrepared.md:54]: Noted

File: api-reference/10 UI Components/dxFilterBuilder/1 Configuration/onEditorPreparing.md (hunk 2 above)
Comment (vladaskorohodova) [reply api-reference/10 UI Components/dxFilterBuilder/1 Configuration/onEditorPreparing.md:62]: Noted

File: concepts/05 UI Components/TextArea/00 Overview.md (hunk 4 above)
Comment (vladaskorohodova) [reply concepts/05 UI Components/TextArea/00 Overview.md:232]: Noted

File: concepts/05 UI Components/TextBox/00 Getting Started with TextBox/10 Configure the Input Mode.md (hunk 5 above)
Comment (vladaskorohodova) [reply concepts/05 UI Components/TextBox/00 Getting Started with TextBox/10 Configure the Input Mode.md]: Noted

File: concepts/05 UI Components/TextBox/00 Getting Started with TextBox/30 Handle the Keyboard Events.md [hunk 9]
```diff
@@ -0,0 +1,90 @@
...
+        }
+    }
+
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxTextBox 
+            v-model:value="value"
+            @enterKey="onEnterKey($event)"
```
Comment (ksercs) [inline concepts/05 UI Components/TextBox/00 Getting Started with TextBox/30 Handle the Keyboard Events.md]:
```suggestion
            @enter-key="onEnterKey($event)"
```

**Review by ksercs — APPROVED** [review]
Code looks OK

File: api-reference/10 UI Components/dxFilterBuilder/1 Configuration/onEditorPrepared.md (hunk 1 above)
Comment (albertov05) [reply api-reference/10 UI Components/dxFilterBuilder/1 Configuration/onEditorPrepared.md:54]: Thanks.

File: api-reference/10 UI Components/dxFilterBuilder/1 Configuration/onEditorPreparing.md (hunk 2 above)
Comment (albertov05) [reply api-reference/10 UI Components/dxFilterBuilder/1 Configuration/onEditorPreparing.md:62]: Thanks.

File: concepts/05 UI Components/TextBox/00 Getting Started with TextBox/10 Configure the Input Mode.md (hunk 5 above)
Comment (albertov05) [reply concepts/05 UI Components/TextBox/00 Getting Started with TextBox/10 Configure the Input Mode.md]: Thanks.

**Pull request merged**